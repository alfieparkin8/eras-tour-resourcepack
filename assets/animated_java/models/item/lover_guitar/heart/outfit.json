{"parent":"animated_java:item/lover_guitar/outfit","textures":{"2":"minecraft:block/outfits/lover/heart_layer_1"}}