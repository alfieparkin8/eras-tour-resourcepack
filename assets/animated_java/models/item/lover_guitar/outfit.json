{
	"textures": {
		"1": "animated_java:item/lover_guitar/sunset_layer_1"
	},
	"elements": [
		{
			"from": [
				3.25,
				18.25,
				6.25
			],
			"to": [
				12.75,
				31.75,
				11.75
			],
			"faces": {
				"north": {
					"uv": [
						5,
						10,
						7,
						16
					],
					"texture": "#1"
				},
				"east": {
					"uv": [
						7,
						10,
						8,
						15.5
					],
					"texture": "#1"
				},
				"south": {
					"uv": [
						8,
						10,
						10,
						16
					],
					"texture": "#1"
				},
				"west": {
					"uv": [
						7,
						10,
						8,
						15.5
					],
					"texture": "#1"
				},
				"up": {
					"uv": [
						7,
						10,
						5,
						8
					],
					"texture": "#1"
				},
				"down": {
					"uv": [
						9,
						10,
						7,
						8
					],
					"texture": "#1"
				}
			}
		}
	],
	"display": {
		"head": {
			"rotation": [
				0,
				180,
				0
			]
		}
	}
}