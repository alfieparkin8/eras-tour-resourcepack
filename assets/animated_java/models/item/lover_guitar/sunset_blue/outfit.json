{
	"parent": "animated_java:item/lover_guitar/outfit",
	"textures": {
		"2": "animated_java:item/lover_guitar/lover_blue"
	}
}