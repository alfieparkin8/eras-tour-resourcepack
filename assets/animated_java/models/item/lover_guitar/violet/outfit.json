{"parent":"animated_java:item/lover_guitar/outfit","textures":{"2":"minecraft:block/outfits/lover/violet_layer_1"}}