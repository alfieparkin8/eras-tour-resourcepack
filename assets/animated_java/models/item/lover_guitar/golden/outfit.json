{"parent":"animated_java:item/lover_guitar/outfit","textures":{"2":"minecraft:block/outfits/lover/golden_layer_1","3":"minecraft:block/outfits/lover/golden_boots_layer_1"}}